koku/api/metrics/constants.py

```diff
@@ -14,12 +14,30 @@
 OCP_METRIC_MEM_GB_EFFECTIVE_USAGE_HOUR = "memory_gb_effective_usage_per_hour"
 OCP_METRIC_STORAGE_GB_USAGE_MONTH = "storage_gb_usage_per_month"
 OCP_METRIC_STORAGE_GB_REQUEST_MONTH = "storage_gb_request_per_month"
+OCP_NODE_CORE_HOUR = "node_core_cost_per_hour"
 OCP_NODE_MONTH = "node_cost_per_month"
+OCP_NODE_CORE_MONTH = "node_core_cost_per_month"
 OCP_CLUSTER_MONTH = "cluster_cost_per_month"
 OCP_PVC_MONTH = "pvc_cost_per_month"
 
-MEMORY_DISTRIBUTION = "memory"
-CPU_DISTRIBUTION = "cpu"
+# defines the usage type for each metric
+CPU = "cpu"
+MEM = "memory"
+STORAGE = "storage"
+METRIC_MAP = {
```

Comment:
Considering we already have `COST_MODEL_METRIC_MAP` in here, it would probably be good to make this var more descriptive. Maybe:
- `METRIC_USAGE_TYPE_MAP`
- `USAGE_METRIC_MAP`

That way there is better clarity between the two metric maps without having to go searching through the code to figure out where it is used.

Reply:
YES!! :+1: